vm_scripts/acquire_ticketid.py

```
@@ -0,0 +1,78 @@
+from pyVmomi import vim
+from .helpers import VmHelper
+from .machines import Machines
+from vm_scripts.helpers import VmHelper
```

Ruff (0.8.2)

Redefinition of unused `VmHelper` from line 2

(F811)